rwengine/src/objects/GameObject.hpp

```diff
@@ -148,6 +148,21 @@ class GameObject {
      */
     void setHeading(float heading);
 
+    /**
+     * @brief getCenterOffset Returns the offset from center of mass to base of model
+     * This function should be overwritten by a derived class
+     */
+    virtual glm::vec3 getCenterOffset() {
+        return glm::vec3(0.f, 0.f, 1.f);
```

Comment:
Because `kSpawnOffset` had an offset of +1.0, and if we later on implement a new object and don't override the function and call `applyOffset` we would expect at least some kind of offset.